cves/CVE-2012-0022.yml

```diff
@@ -103,8 +105,8 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
+  answer: false
+  code: false
   fix:
```

Comment:
The code and fix fields are supposed to contain the boolean indicators, while the answer field is intended to be a shortform summary of what tests you found or where you looked for tests if you didn't find any.